packages/api/resolvers/types/bookmark.ts

```diff
@@ -0,0 +1,11 @@
+import { Users } from 'meteor/unchained:core-users';
+import { Products } from 'meteor/unchained:core-products';
+
+export default {
+  product(obj) {
+    return Products.findOne({ _id: obj.productId });
```

Comment:
Shouldn't you also use `modules.bookmarks...` here?

Reply:
To find products?